fastjet/internal/plot/delaunay_test.go

```diff
@@ -0,0 +1,72 @@
+// Copyright 2017 The go-hep Authors.  All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package plot
+
+import (
+	"bytes"
+	"image"
+	"io/ioutil"
+	"os"
+	"reflect"
+	"testing"
+
+	"go-hep.org/x/hep/fastjet/internal/delaunay"
+)
+
+func TestDelaunayPlot(t *testing.T) {
+	fname := "delaunay_example.png"
+	ref := "testdata/delaunay_example.png"
+	err := DelaunayPlotExample(fname)
+	if err != nil {
+		t.Fatal(err)
+	}
+	want, err := ioutil.ReadFile(ref)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	got, err := ioutil.ReadFile(fname)
+	if err != nil {
+		t.Fatal(err)
+	}
+	v1, _, err := image.Decode(bytes.NewReader(want))
+	if err != nil {
+		t.Fatal(err)
+	}
+	v2, _, err := image.Decode(bytes.NewReader(got))
+	if err != nil {
+		t.Fatal(err)
+	}
+	equal := reflect.DeepEqual(v1, v2)
+	if !equal {
+		t.Errorf("error: %s differs from reference file", fname)
```

Comment:
use `t.Fatalf` instead, otherwise, the test file will be removed, preventing somebody to investigate w/o modifying the code.

Reply:
done